packages/jaeger-ui/src/middlewares/index.js

```diff
@@ -70,7 +70,7 @@ export const loadJsonTracesMiddleware = store => next => action => {
   } else {
     return next(action);
   }
-  return next(action); // Consistent returns
+  return undefined;
```

Comment:
why is this the right behavior?

Reply:
I assume calling the next action is a pattern that needs to be maintained, it looks like a dangerous shortcut when you change it just to work around another problem. The right fix is to control the order of these middlewares, so that OTLP translator is invoked before the rest of the handlers.